resources/helpers.js

```diff
@@ -27,6 +27,8 @@ const modelShareRegEx = /https:\/\/modelshare\.mendixcloud\.com/;
 const forumRegEx = /https:\/\/forum\.mendixcloud\.com/;
 const supportRegEx = /https:\/\/support\.mendix\.com/;
 const dataHubRegEx = /https:\/\/hub(-test|-ofdata)?\.mendixcloud\.com/;
+// We respect the URL given by the team, so we use all-lower-case here.
+const privatecloudRegEx = /https:\/\/privatecloud(-test|-accp)\.mendixcloud\.com/;
```

Comment:
Can you please change this to `https:\/\/privatecloud(-test|-accp)?\.mendixcloud\.com`?
Our production environment is https://privatecloud.mendixcloud.com

Reply:
Ah, thanks. Indeed there's a missing `?`